drivers/tee/optee/smc_abi.c

```diff
@@ -865,13 +952,15 @@ static void optee_handle_rpc(struct tee_context *ctx,
  * Returns return code from secure world, 0 is OK
  */
 static int optee_smc_do_call_with_arg(struct tee_context *ctx,
-				      struct tee_shm *shm, u_int offs)
+				      struct tee_shm *shm, u_int offs,
+				      bool system_call)
```

Comment:
System call in kernel context is will make most kernel developers think of something different, how about `system_thread` or `sys_thread`?